deploy/operator/operator.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nmstate-operator
+  namespace: {{ .OperatorNamespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: nmstate-operator
+  template:
+    metadata:
+      labels:
+        app: kubernetes-nmstate
+        name: nmstate-operator
+    spec:
+      serviceAccountName: nmstate-operator
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule
+      containers:
+        - name: nmstate-operator
+          args:
+          - --v=production
+          # Replace this with the built image name
+          image: {{ .NMStateOperatorImage }}
+          imagePullPolicy: {{ .OperatorPullPolicy }}
+          command:
+          - kubernetes-nmstate
+          env:
+            - name: WATCH_NAMESPACE
+              value: ""
+            - name: OPERATOR_NAME
+              value: "kubernetes-nmstate-operator"
+            - name: ENABLE_PROFILER
+              value: "False"
+            - name: PROFILER_PORT
+              value: "6060"
+            - name: RUN_OPERATOR
+              value: ""
+            - name: HANDLER_IMAGE
+              value: {{ .NMStateHandlerImage }}
```

Comment:
Can we make it `HandlerImage` for consistency ( I have do same comment on prefix PR )